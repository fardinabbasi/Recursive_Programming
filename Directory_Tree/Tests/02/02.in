6 2
folder1 0
file1 1
folder2 1
file2 3
folder3 3
file3 5