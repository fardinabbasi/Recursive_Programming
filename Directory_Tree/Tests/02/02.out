|__folder1
   |__file1
   |__folder2